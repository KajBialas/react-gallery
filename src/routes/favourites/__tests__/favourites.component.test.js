import React from 'react';
import { shallow } from 'enzyme';
import FavouritesComponent from '../favourites.component';

describe('FavouritesComponent', () => {
  it('should shallows correctly', () => {
    const component = shallow(<FavouritesComponent />);

    expect(component).toMatchSnapshot();
  });
});