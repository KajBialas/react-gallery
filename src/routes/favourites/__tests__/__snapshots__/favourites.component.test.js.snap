// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavouritesComponent should shallows correctly 1`] = `
<Fragment>
  <HeaderComponent />
  <styled.div>
    <styled.div>
      My 
      <span>
        favourites images
      </span>
    </styled.div>
    <GalleryComponent
      addFavourite={[Function]}
      favouritesRecords={Array []}
      handleImagesLoaded={[Function]}
      photosRecords={Array []}
      removeFavourite={[Function]}
    />
  </styled.div>
</Fragment>
`;
